amundsen_application/static/css/_avatars.scss

```diff
@@ -1,7 +1,3 @@
-.sb-avatar > div {
```

Comment:
I would only want to apply this in the `NavBar` from the perspective that the `NavBar` usage is the only one I'm aware was discussed, and the white border is useful in other places like in the FrequentUsers UI where we stack these on top of one another.

Reply:
I see, I will retrieve it for the FrequentUsers UI.